src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/IssueNumberRetriever.java

```diff
@@ -23,7 +27,18 @@ public IssueNumberRetriever(DataSource dataSource) {
     public long getNextIssueNumber() {
         try {
             Statement statement = dataSource.getConnection().createStatement();
-            ResultSet resultSet = statement.executeQuery(GET_NEXTVALUE_FROM_ISSUE_NUMBER_SEQUENCE);
+            ResultSet resultSet = statement.executeQuery(GET_NEXT_VALUE_FROM_ISSUE_NUMBER_SEQUENCE);
```

Comment:
Duplicated code. 'getNextIssueNumber' and 'getCurrentIssueNumber' are almost the same, they differ only with SQL query.  

+1 for JdbcTemplate